projects/1.16/assets/more-bows-restrung/more-bows-restrung/lang/zh_cn.json

```
@@ -0,0 +1,32 @@
+{
+    "item.morebows.diamond_bow": "钻石弓",
+    "item.morebows.ender_bow": "末影弓",
+    "item.morebows.flame_bow": "烈焰弓",
+    "item.morebows.frost_bow": "寒霜弓",
+    "item.morebows.gold_bow": "黄金弓",
+    "item.morebows.iron_bow": "铁质弓",
+    "item.morebows.multi_bow": "双发弓",
+    "item.morebows.stone_bow": "强化弓",
+    "morebows.confBowDamageMult": "伤害倍数",
+    "morebows.confBowDamageMult.tooltip": "弓的基础伤害倍数越高伤害越大。",
+    "morebows.confBowDrawbackDiv": "攻击间隔",
+    "morebows.confBowDrawbackDiv.tooltip": "弓的攻击间隔，越小攻速越快.",
+    "morebows.confBowDurability": "耐久性",
+    "morebows.confBowDurability.tooltip": "弓的耐久值，越高越耐用.",
+    "morebows.confCatBow": "弓属性",
+    "morebows.confCatBow.tooltip": "允许改变每把弓的属性。对整合包作者有用。",
+    "morebows.confCatGen": "通用设置",
+    "morebows.confCatGen.tooltip": "杂项",
+    "morebows.confGenFrostCold": "冰霜箭矢是寒冷的",
+    "morebows.confGenFrostCold.tooltip": "如果启用，冰霜箭将对火属性生灵造成三倍伤害，并灭火。\n启用这个也可以防止冰霜箭着火。",
+    "morebows.confGenOldRendering": "旧的冰霜箭矢渲染",
+    "morebows.confGenOldRendering.tooltip": "如果启用了这个选项，冰霜箭将被渲染成箭头。",
+    "morebows.confGenOldSlowdown": "冰霜箭矢减速",
+    "morebows.confGenOldSlowdown.tooltip": "如果启用，冰霜箭矢将会通过将实体放入一个网中1tick的方法减速实体\n如果禁用，冰霜箭矢会在命中时使用等同于减速药剂的效果减速实体.",
+    "morebows.confMultiShotAmmo": "多重射击箭矢使用",
+    "morebows.confMultiShotAmmo.AlwaysCustomArrows": "使用一支特殊箭矢射击",
+    "morebows.confMultiShotAmmo.AlwaysStandardArrows": "使用一支标准箭矢射击",
+    "morebows.confMultiShotAmmo.UseAmountShot": "使用尽可能多的箭射击",
+    "morebows.confMultiShotAmmo.tooltip": "改变多发弓发射额外箭的方式。可选的选项:\n使用一支箭，发射多支特殊箭\n使用一支箭，射出多支普通箭\n使用尽可能多的箭，射出用过的箭头",

```

```suggestion
    "morebows.confMultiShotAmmo.tooltip": "改变多发弓发射额外箭的方式。选项如下：\n使用一支箭，射出多支特殊箭\n使用一支箭，射出多支普通箭\n使用尽可能多的箭，射出用过的箭",
```
最后的used arrow在游戏中实际表现如何